src/CarouselItem.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+use Yiisoft\Html\Html;
+use Yiisoft\Html\Tag\Img;
+
+/**
+ * Represents a carousel item.
+ */
+final class CarouselItem
+{
+    public function __construct(
+        private readonly Img $image,
```

Comment:
From docs: "It works with a series of images, text, or custom markup."

Suggest use `Stringable|string` instead `Img` type.

Reply:
Could you show me the example of the documentation, to guide me when it is text or any markup, thks.

Reply:
I will add two tests, for both cases, they are possible.

Reply:
done.